porting/npl/riot/include/nimble/nimble_npl_os.h

```diff
@@ -259,7 +262,9 @@ ble_npl_time_ticks_to_ms32(ble_npl_time_t ticks)
 static inline void
 ble_npl_time_delay(ble_npl_time_t ticks)
 {
-    xtimer_usleep(ticks * 1000);
+    uint64_t us64 = ticks * US_PER_MS;
+    assert(us64 <= UINT32_MAX);
+    xtimer_usleep(us64);
```

Comment:
getting RIOT ready was easier than expected: https://github.com/RIOT-OS/RIOT/pull/13370

So we can drop the assert again and use `xtimer_usleep64()` directly here.

Reply:
> So we can drop the assert again and use `xtimer_usleep64()` directly here.

Done!